tools/bumper/component_commands.go

```diff
@@ -216,14 +215,21 @@ func (componentOps *gitComponent) getLatestTaggedFromBranch(repo, owner, branch,
 			return "", "", errors.Wrap(err, "Failed to get commit-sha from tag name")
 		}
 
-		for _, commit := range branchCommits {
-			if commit.GetSHA() == commitShaOfTag {
-				return tagName, commit.GetSHA(), nil
+		for pageIdx := 0; pageIdx < maxPagePaginate; pageIdx++ {
```

Comment:
i think first page is 1 maybe according 
https://docs.github.com/en/rest/commits/commits?apiVersion=2022-11-28#list-commits

Reply:
if you want, you can start without saying the page first time (even have it outside the loop as it was)
and only upon needs have it in the loop (not overriding the outer one)
and use there next_page field to determine the next page

Reply:
you're right. Fixed